src/js/app/controllers.js

```diff
@@ -10,12 +10,12 @@ require('./controllers/post/PostCtrl')(angularModule)
 require('./controllers/post/PostListCtrl')(angularModule)
 require('./controllers/post/CommentsCtrl')(angularModule)
 
-require('./controllers/community/CommunityCtrl')(angularModule)
-require('./controllers/community/CommunitySettingsCtrl')(angularModule)
-require('./controllers/community/CommunityPostsCtrl')(angularModule)
-require('./controllers/community/CommunityMembersCtrl')(angularModule)
-require('./controllers/community/CommunityInviteCtrl')(angularModule)
-require('./controllers/community/NewCommunityCtrl')(angularModule)
+require('./controllers/community/CommunityCtrl')(angularModule);
```

Comment:
this file looks like it had some accidental overwrites during merging -- 1) semicolons here